docs/devel/e2e-node-tests.md

```diff
@@ -63,30 +63,40 @@ and remote passwordless sudo access over ssh)
 * using [run_e2e.go](../../test/e2e_node/runner/run_e2e.go) to build a tar.gz
 and executing on host (requires host access w/ remote sudo)
 
-### Configuring a new remote host for testing
+### Option 1: Configuring a new remote host from scratch for testing
 
-The host must contain a environment capable of supporting a mini-kubernetes
-cluster. Includes:
+The host must contain an environment capable of running a minimal kubernetes cluster
+consisting of etcd, the kube-apiserver, and kubelet. The steps required to step a host vary between distributions
+(coreos, rhel, ubuntu, etc), but may include:
 * install etcd
 * install docker
+* add user running tests to docker group
 * install lxc and update grub commandline
 * enable tty-less sudo access
 
-See [setup_host.sh](../../test/e2e_node/environment/setup_host.sh)
+These steps should be captured in [setup_host.sh](../../test/e2e_node/environment/setup_host.sh)
 
-### Running the tests
+### Option 2: Copying an existing host image from another project
```

Comment:
Nope.  I will need to update the instructions again with the new process once all of the update PRs are in.  Plan on just waiting until the Makefile rule is in and updating the instructions with the new process.